src/Microsoft.DotNet.Interactive.Formatting/Formatter.cs

```diff
@@ -403,13 +392,12 @@ public static void Register(ITypeFormatter formatter)
         /// <param name="formatter">The formatter.</param>
         /// <param name="mimeType">The MimeType for this formatter. If it is not specified it defaults to <see cref="PlainTextFormatter.MimeType"/></param>
         public static void Register<T>(
-            Func<FormatContext, T, TextWriter, bool> formatter,
+            FormatDelegate<T> formatter,
```

Comment:
We need to inform external contributors, this will break XPlot.Plotly and Plotly.NET.Interactive as we merge. Is it possible to retain the old Register and label as deprecated?

Reply:
Do you know that they were using this overload? We can check easily enough.

Reply:
this one for example [FormatterKernelExtension](https://github.com/plotly/Plotly.NET/blob/63cadbe361fea10c0dcd585ca78a2d08c682a178/src/Plotly.NET.Interactive/Extension.fs)

Reply:
And [this](https://github.com/pi-top/pi-top-4-.NET-SDK/blob/master/src/PiTop.Camera.InteractiveExtension/KernelExtension.cs) is the code in pi-top

Reply:
That's using a different overload.